torch/_ops.py

```diff
@@ -65,6 +65,15 @@ def overloadpacket(self):
     def op(self):
         return self._op
 
+    def impl(self, lib_obj, dispatch_key, fn):
+        name = self.__name__ if self._overloadname != 'default' else self.__name__.split(".")[0]
+        lib_obj.impl(name, dispatch_key, fn)
+        return
```

Comment:
We were going to get rid of this helper right?

Reply:
nit: trailing return is unnecessary

Reply:
yeah we are